antiplag/tests/test_intervals.py

```diff
@@ -6,6 +6,7 @@
 
 class ResultModelTestCase(TestCase):
     def setUp(self):
+        self.maxDiff = None
```

Comment:
Just curious, does it work on this project? I've tried it before, but with no luck.

Reply:
Yes, happened to me - when I had failing test, it showed up like FooLoremInput[234 more chars]EndingSequence.

When I used this option, the whole log was printed.